pkg/services/rediscache/plan_detail.go

```diff
@@ -0,0 +1,33 @@
+package rediscache
+
+const basic = "basic"
+const standard = "standard"
+const premium = "premium"
+
+type planDetail struct {
```

Comment:
i wasn't questioning the definition of the struct, just those methods below that a.) aren't exported and b.) are only called from catalog.go..unless I'm missing a usage.